sycl/source/queue.cpp

```diff
@@ -190,6 +190,11 @@ queue::has_property<property::queue::enable_profiling>() const;
 template __SYCL_EXPORT property::queue::enable_profiling
 queue::get_property<property::queue::enable_profiling>() const;
 
+template __SYCL_EXPORT bool
+queue::has_property<property::queue::in_order>() const;
+template __SYCL_EXPORT property::queue::in_order
+queue::get_property<property::queue::in_order>() const;
```

Comment:
I guess, this is enough for instantiating a function specialization and the function template is actually defined in headers. Hence, what's the reason for having these symbols exported?

Reply:
Without this specializations code like "Queue1.get_property<property::queue::in_order>()" will not correctly link.